packages/test/test-end-to-end-tests/src/test/idCompressor.spec.ts

```diff
@@ -979,15 +995,15 @@ describeCompat("IdCompressor Summaries", "NoCompat", (getTestObjectProvider) =>
 		// Check directly that ID compressor is issuing short IDs!
 		// If it does not, the rest of the tests would fail - this helps isolate where the bug is.
 		const idTest = defaultDataStore._context.containerRuntime.generateDocumentUniqueId();
-		assert(typeof idTest === "number", "short IDs should be issued");
-		assert(idTest >= 0, "finalId");
+		assertInvert(typeof idTest === "number", "short IDs should be issued");
+		// assert(idTest >= 0, "finalId");
```

Comment:
Maybe be roll it into previous assert? I.e. 

assertInvert(typeof idTest === "number" && idTest >= 0, "short IDs should be issued");

Reply:
Done